content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -85,11 +85,17 @@ Datadog consolidates and applies additional tags from various sources to cost me
 
 ### Containers
 
-| Out-of-the-box tag  |  Description |
-| ---                 | ------------ |
-| `orchestrator`      | The orchestration platform associated with the item (kubernetes, ecs). |
-| `allocated_spend_type`     | The spend category associated with the cost. Cluster costs are allocated into three spend types: resources used by a workload (`usage`); resources reserved by a workload, but not used (`workload_idle`); and resources that are not reserved or used by any workload (`cluster_idle`). *Only available for `.shared.resources.allocated` metrics.* |
-| `allocated_resource`   | The resource category associated with the item (cpu, memory). *Only available for `.shared.resources.allocated` metrics.* |
+| Out-of-the-box tag                         | Description                                                                                                                                                   |
+| ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------  |
+| `orchestrator:kubernetes`                  | The orchestration platform associated with the item is Kubernetes.                                                                                            |
+| `orchestrator:ecs`                         | The orchestration platform associated with the item is ECS.                                                                                                   |
+| `allocated_spend_type:usage`               | The spend category associated with resources costs used by a workload. *Only available for `.shared.resources.allocated` metrics.*                            |
```

Comment:
a bit of a mouthful, not sure "spend category" really helps... what do we think of just:
`Cost of resources used by a workload`
`Cost of resources reserved and allocated by a workload, but not used`
`Cost of resources that are not reserved or used by any workload`
...
`Cost of CPU resources`
`Cost of memory resources`

etc.

Reply:
Updated, feel free to take another look.